src/components/EventCard/index.jsx

```diff
@@ -1,38 +1,54 @@
 import { faBookmark as notBookmarked } from '@fortawesome/free-regular-svg-icons';
 import { faBookmark } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import moment from 'moment';
 import PropTypes from 'prop-types';
 import { useState } from 'react';
 import styles from './event-card.module.css';
 
-const imageUrl = `https://s3-alpha-sig.figma.com/img/699d/1183/8d9716e61a45f9e95043b56334473c19?Expires=1714348800&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=RZ4xRYD3OvMQM-9CYNL9jNLYbACRF-T7yDNBW904td9Y3hw6WnzRARE8iebdaW0X930VG0s~eD2FQ1h4DCy2Oyly9ajQBeGkpdN1XR2zXRCaVt~n7NxYtdEORMyqYq18Q6q1a6jEUGdnxBtA2Z22UaPKHK4BxrifO3PYCqr6MTLMnmggWE3VnSVBu751JDB2WwyUKbV~vLOmFdoeeWENxxirrIP-3d6agV5darB9BgxV31VawN6DNPJ8YKYSg4vURbetFmbTJCxmHkkbq8TnfKgUwxs3x0aLmo8Op1BtBLNcz7rPc8fgZGQZFBkQS~6U8hZ1n3E1w66OIELE4Mn7kA__`;
-
 function EventCard({ event }) {
   const [isBookmarked, setIsBookmarked] = useState(false);
 
   return (
     <article className={styles.card}>
       <div
         className={styles.bookmark}
-        onClick={() => setIsBookmarked(!isBookmarked)}
+        onClick={() => {
```

Comment:
I think a more efficient way of doing this will be to include `isBookmarked` as a property in the event object. This way, we get rid of the internal state and simply use the value of the property in the event object to determine what events have been bookmarked by a user.

Reply:
Managing events bookmarked from the backend by users would require a new table, new routes and all. That just doesn't seem like a good idea to me, especially with the time constraint. Hence why bookmarks are being persisted on the frontend instead. We just need to clean it up and most likely use a context container so it's much cleaner and streamlined.

Reply:
Moving the bookmarks to context/state will not solve the issue. The problem is, the `EventCard` component displays only events from its `event` prop. And the value of that prop is coming from the `Browse` page - the events from the database. The only way I think we can achieve this functionality from the frontend is to manually add an `isBookmarked` property to the events gotten from the database and saving it to local storage. We then do a comparison of both arrays(local storage and database) each time the page is loaded, and add the field to the new and existing events before we then display it on the UI. Tbh, I don't think it's a great approach.

Reply:
- Doesn't need to be complicated, since we don't want to handle the bookmarked events at the backend, we can just follow @ezeaniiandrew suggestion. The bookmarked events will only be cleared when the user clear the local storage.

Reply:
Resolved. Kindly review and approve.